src/main/java/org/desarrolladorslp/technovation/services/impl/MessageServiceImpl.java

```diff
@@ -34,33 +43,67 @@
 
     @Override
     @Transactional
```

Comment:
Transactional for `getSpecificMessageByUser` can be marked as `(readOnly = true)`